starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,46 @@
 // Names and Input
+var hacker1 = "Arnaud";
 
+console.log("driver's name is" + hacker1);
+
+let hacker2 = prompt("What's your name?");
+console.log("navigator's name is" + hacker2);
 
```

Comment:
Don't forget the space in your console log(navigator's name is (HERE)" + hacker2), otherwise you will get "navigator's name ishacker2"